catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitorIT.java

```diff
@@ -123,6 +159,27 @@ public void testInPlaceMonitoring()
     }
 
     @Test
+    public void testInPlaceDavMonitoring()
+            throws IOException, InterruptedException, SourceUnavailableException, IngestException,
+            MimeTypeParseException, StorageException, URISyntaxException {
+        updateContentDirectoryMonitor("http://localhost:" + stubServer.getPort() + "/webdavtest",
```

Comment:
Is the stubServer port a fixed port or is it the first available port? Just want to make sure we don't run into port conflicts.